spec/features/admin/rebate_forms/show_spec.rb

```diff
@@ -50,6 +50,16 @@
           expect(page).to have_text('Signature required')
         end
       end
+
+      context 'when the generate QR button is clicked' do
+        it 'goes to the right place' do
+          visit "/admin/rebate_forms/#{rebate_form.id}/"
+          click_link('Generate QR')
```

Comment:
clicking the button should just display the QR code but not actually navigate away from the page, shouldn't it?

Reply:
it is served from its own route